pipelines/pipeline.go

```diff
@@ -0,0 +1,84 @@
+package pipelines
+
+import (
+	"encoding/binary"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+	"hash/fnv"
+	"math/rand"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+type Pipeline struct {
```

Comment:
Is there any way we can change either the package name or this struct name? Trying to avoid `pipelines.Pipeline`

Reply:
It would be good yes, I don't see great options though, I can think of `pipelines.Context`, `pipelines.State`? Do you have other ideas?